collection-scripts/gather

```diff
@@ -38,6 +40,7 @@ elif [ "${reconcileStrategy}" = "standalone" ]; then
 else
     dbglog "Collecting entire ceph logs"
     gather_ceph_resources ${BASE_COLLECTION_PATH}
+    gather_odf_consumer ${BASE_COLLECTION_PATH}
```

Comment:
- does above command `ceph_resources` fail silently
- the log will say `ceph logs` but you'll not access ceph via cli at consumer end
- btw, let's use the naming `client` inplace of `consumer`

Reply:
Yes ceph_resources will fail silently.
renamed client with consumer